webui/react/src/pages/F_ExpList/glide-table/MultiSortMenu.tsx

```diff
@@ -151,6 +151,7 @@ const DirectionOptions: React.FC<DirectionOptionsProps> = ({ onChange, type, val
 const ColumnOptions: React.FC<ColumnOptionsProps> = ({ onChange, columns, value }) => (
   <Select
     autoFocus
+    dropdownMatchSelectWidth={300}
```

Comment:
Could we do the same thing to filter dropdown too?

Reply:
it is done in filter from the beginning :)
or do you want me to increase the width size in filter?

Reply:
Can they use the same width to look more uniform?

Reply:
ok made both 300px width